spec/models/registration_spec.rb

```diff
@@ -22,4 +22,28 @@
       end
     end
   end
+
+  describe '#destroy_purchased_tickets' do
+    it 'destroys purchased tickets if tickets are purchased' do
+      user = build(:user)
+      conference = build(:conference)
```

Comment:
Can you please move this line and line 13 in a ``let(:conference) ...`` somewhere at the beginning of the file?